ds-css/webpack.config.js

```diff
@@ -8,7 +8,7 @@ const StylelintPlugin = require('stylelint-webpack-plugin');
 module.exports = {
   performance: {
     hints: "error", // enum
-    maxAssetSize: 200000, // int (in bytes),
+    maxAssetSize: 2000000, // int (in bytes),** Increased this by adding a 0 temporarily **
```

Comment:
let's come back to this as discussed and review the stylelint rules, also we should verify the final css

Reply:
I ran npm run build and the final file size was 192kb. Before I made the change it threw an error.


<img width="693" alt="Screen Shot 2020-03-23 at 4 22 37 PM" src="https://user-images.githubusercontent.com/1970478/77359875-b28b2300-6d22-11ea-9107-d9e7856c1f3a.png">